pubspec.yaml

```diff
@@ -4,8 +4,8 @@ version: 0.2.0
 homepage: https://github.com/letsar/value_layout_builder
 
 environment:
-  sdk: ">=2.7.0 <3.0.0"
-  flutter: ">=1.20.0 <2.0.0"
+  sdk: ">=2.12.0-0 <3.0.0"
```

Comment:
We should remove the -0 suffix:
```yaml
environment:
  sdk: '>=2.12.0 <3.0.0'
```

Reply:
I think it's not possible it's gave me the error 
`Because value_layout_builder requires SDK version >=2.12.0 <3.0.0, version solving failed.`
when I tried to `pub get`

Reply:
oups it's a error cause by the preview channel of flutter